docs/templating/design_engine/design_engine.md

```diff
@@ -35,8 +35,8 @@ Each scope can use only one design.
 ## Order of themes
 
 The order of themes in a design is important.
-The design engine attempts to apply the first theme in configuration (for example, `theme1`).
-If it cannot find the required template or asset in this theme, it proceeds to the next theme in the list (for example, `theme2`).
+The design engine attempts to apply the first theme in configuration (for example, `theme2`).
+If it cannot find the required template or asset in this theme, it proceeds to the next theme in the list (for example, `theme1` then `theme0` and finally the default `standard`).
```

Comment:
From my POV, it still misses a link between a design's theme list, `@ibexadesign` in paths, and the templates/themes/ directories.

Reply:
Proposed in 5d316481533122bb2705d7f4e84537ef083b83ac